### ISSUE TITLE
Webpack 4 support

### ISSUE THREAD

**Issue by asual** [chipto/css-visor#4]
Using the plugin with the latest Webpack results in the following error:

```
node_modules/css-visor/plugin.js:40
                    if (chunk.isInitial()) {
                              ^
TypeError: chunk.isInitial is not a function
    at compilation.chunks.forEach.chunk
```

**Comment by zikaari** [comment]
Mind sharing your `package.json`?

If you can't for legal reasons, then just the dependencies section and start script?
Thanks

**Comment by asual** [comment]
Here is the most minimal working package.json I was able to come up with:

```json
{
    "name": "css-visor",
    "version": "0.1.0",
    "private": true,
    "scripts": {
        "start": "webpack-dev-server"
    },
    "devDependencies": {
        "css-loader": "0.28.10",
        "css-visor": "1.0.0",
        "extract-text-webpack-plugin": "3.0.0",
        "html-webpack-plugin": "2.30.1",
        "webpack": "3.11.0",
        "webpack-cli": "2.0.9",
        "webpack-dev-server": "2.11.2"
    }
}
```

The originally reported error pops up when the following package versions are updated:

```
    "extract-text-webpack-plugin": "4.0.0-alpha.0",
    "html-webpack-plugin": "webpack-contrib/html-webpack-plugin",
    "webpack": "4.0.1",
    "webpack-dev-server": "3.0.0"
```

The webpack.config.js is the following:

```js
const CSSVisor = require('css-visor');
const HtmlWebpackPlugin = require('html-webpack-plugin');

module.exports = {
    entry: "./index.js",
    mode: "development", // Only needed for Webpack 4
    module: {
        rules: [
            {
                test: /\.css$/,
                use: [
                    {
                        loader: "css-visor/loader"
                    },
                    {
                        loader: "css-loader",
                        options: {
                            sourceMap: true,
                        },
                    },
                ],
            },
        ],
    },
    output: {
        publicPath: "/",
    },
    plugins: [
        new HtmlWebpackPlugin({
            filename: "index.html",
            inject: true,
            template: "index.ejs",
        }),
        new CSSVisor(),
    ],
};
```

**Comment by zikaari** [comment]
Perfect!

[`ee26ac3`](https://github.com/chipto/css-visor/commit/ee26ac34822d4b805c8e78c470c5b16f692d7458 ) should fix this, available in `v2.0.0`

Thanks for reporting!

Please close this issue if you believe the problem is resolved.

**Comment by zikaari** [comment]
Closing due to inactivity
